tests/functional/adapter/test_provision_users.py

```diff
@@ -13,13 +13,13 @@
             select *
             from sys.database_principals
             where name = '{{ env_var('DBT_TEST_AAD_PRINCIPAL_1') }}')
-        drop user {{ env_var('DBT_TEST_AAD_PRINCIPAL_1') }}
+        drop user [{{ env_var('DBT_TEST_AAD_PRINCIPAL_1') }}]
```

Comment:
@sdebruyn argh, why is the test not using my updated definition?
https://github.com/dbt-msft/dbt-sqlserver/actions/runs/4961293264/jobs/8877902126#step:8:2365

Reply:
it does not make sense, locally it works